site/plugins.json

```
@@ -437,6 +437,14 @@
     "name": "Gmail",
     "package": "netlify-plugin-gmail",
     "repo": "https://github.com/racoonx2p/netlify-plugin-gmail",
-    "version": "1.1.0"
+    "version": "1.1.0",
```

```suggestion
    "version": "1.1.0"
```

JSON does not allow trailing commas.